jobs/reset.js

```diff
@@ -4,28 +4,107 @@ const dbs = require('../lib/dbs')
 const githubQueue = require('../lib/github-queue')
 const { createDocs } = require('../lib/repository-docs')
 
-module.exports = async function ({ repositoryFullName }) {
+module.exports = async function ({ repositoryFullName, accountId }) {
   repositoryFullName = repositoryFullName.toLowerCase()
```

Comment:
This will throw an error if `repositoryFullName` isn‘t passed in, and in general the function would fail at various later points too in that case, maybe that would be nice to catch in advance.